packages/react-components/react-tooltip/package.json

```diff
@@ -41,7 +41,8 @@
     "@fluentui/react-theme": "^9.1.15",
     "@fluentui/react-utilities": "^9.15.1",
     "@griffel/react": "^1.5.14",
-    "@swc/helpers": "^0.5.1"
+    "@swc/helpers": "^0.5.1",
+    "keyborg": "^2.1.0"
```

Comment:
wondering if this the right pattern to follow ? cc @ling1726 

is it ok to access low level keyborg directly ? AFAIK we should stick to react-tabster (because it abstracts keyborg/tabster breaking changes )

Reply:
I don't think there is any 'abstracting' happening, but rather keeping our dependency graph cleaner. If keyborg/tabster go under a major bump that is isolated to one single package in our ecosystem instead of multiple packages.

I don't think it's too much work to re-export the constant and the event type from react-tabster. Preferrably mark them with `@internal` to discourage users somewhat to use them - even if we commit to not breaking on our end